code/modules/clothing/suits/hoodies.dm

```diff
@@ -174,3 +174,69 @@
 	name = "space carp hood"
 	desc = "A hood attached to a space carp costume."
 	icon_state = "carp_helm"
+
+/obj/item/clothing/suit/storage/hooded/wintercoat/hoodie
+	name = "hoodie"
+	desc = "Warm and cozy."
+	icon_state = "hoodie"
+	var/icon_open = "hoodie_open"
+	var/icon_closed = "hoodie"
+	item_state = "hoodie"
+	hoodtype = /obj/item/clothing/head/winterhood/hoodie
+	/obj/item/clothing/suit/storage/hooded/wintercoat/hoodie/RemoveHood()
+		..()
+		icon_open = "[initial(icon_open)]"
+		icon_closed = "[initial(icon_closed)]"
```

Comment:
Is there a need to encapsulate initial inside string template?

Reply:
How can this be done alternatively? I just referenced how RemoveHood() works, but adjusted it for these icon states.

Reply:
```DM
icon_open = initial(icon_open)
icon_closed = initial(icon_closed)```

Reply:
What's the difference?